### PR TITLE
Added Page Scroll Up Button

### PR DESCRIPTION
**Pull request by mgjain9871**
Before:
image

After:
image

### CODE REVIEW

**Comment by mgjain9871** [comment]
Please consider it under Hacktoberfest
@Syknapse

**Comment by mgjain9871** [comment]
Hey @CandidDeer, please close the pull request with the hacktoberfest acceptes label if everything fits well.

**Comment by mgjain9871** [comment]
Hey @TenTraicion , please close this pull reques with issue #1804 with the hacktoberfest-accepted label

**Comment by TenTraicion** [comment]
> Hey @TenTraicion , please close this pull reques with issue #1804 with the hacktoberfest-accepted label

@mgjain9871 kindly wait for actual reviewer. Thanks!

**Comment by Syknapse** [comment]
hi @mgjain9871 thank you very much for this contribution. It really adds a good functionality and improves the website. We appreciate it. I will go ahead now and start a code review. Unfortunately, even though the button works correctly, but since the code has been copied from somewhere else you will need to do a fair bit of refactoring before we can merge it. I will let you know exactly what needs changing in the review. Thanks again for this

File: assets/script.js
Comment (Syknapse) [inline assets/script.js:128]: You do not need the document ready block. You can remove the two functions inside of it outside and delete this line and it's corresponding closing tags

File: assets/script.js
Comment (Syknapse) [inline assets/script.js:144]: This entire block needs to be deleted. `menu`is not defined and it is causing a JS error. And this block is not doing anything here. Remove it please from line 144 to 147.

File: assets/style.css
Comment (Syknapse) [inline assets/style.css:27]: Remove this padding and replace it with `width: 42px; height: 42px` which will maintain the same size but with actual dimensions and not just padding

File: assets/style.css
Comment (Syknapse) [inline assets/style.css:32]: You should remove all the remaining CSS you added here, from this line 32 to line 63. None of it is doing anything at all, and there are vars that aren't even declared in this project. The only thing you need from all of this section is the background in `.cd-top` which is needed for the arrow. Move it above to the main element styles `#toTopBtn`
```css
background: url(https://res.cloudinary.com/dxfq3iotg/image/upload/v1571057658/cd-top-arrow.svg) no-repeat center 50%;
```

File: index.html
Comment (Syknapse) [inline index.html:2232]: please remove all the `class` and the `data`attributes, they are not being used for anything. You need to keep only `href` and `id`

**Review by Syknapse — CHANGES_REQUESTED** [review]
Your code review is ready, please make the changes requested and we will happily merge this and tag it with hacktoberfest-accepted. Thank you

**Comment by TenTraicion** [comment]
@mgjain9871, kindly make the requested changes so that we can merge your PR smoothly. for any kind of query you can ask here or in our [Discord Community: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF)

**Comment by Syknapse** [comment]
Since the contrinutor has not made the changes requested but this can be a useful addition to our web, I would encourage any of our maintainers to make the requested changes themselves. This way we can merge it and benefit from this enhancement

**Comment by JAGDF** [comment]
Wow I wonder how you code the scroll-up function

**Comment by win21H2** [comment]
Hello @mgjain9871,
Since we already have a scroll-to-top function on the page, I will close this PR.

Thanks, 
win21H2
324Hz

**Pull request closed without merging**
